packages/gatsby/src/redux/machines/page-component.ts

```diff
@@ -124,14 +131,17 @@ module.exports = Machine(
         },
       }),
       deletePage: assign({
-        pages: (ctx, event) => {
+        pages: (ctx, event: AnyEventObject) => {
           ctx.pages.delete(event.page.path)
           return ctx.pages
         },
       }),
       setBootstrapFinished: assign({
-        isInBootstrap: false,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        isInBootstrap: (_ctx, _event): boolean => false,
       }),
     },
   }
 )
+
+export default compoenentMachine
```

Comment:
Let's use named export adhering to guidelines
> Only use named exports, no export default.
 it makes CommonJS and ES easer to interop